roles/oneview_label_facts/README.md

```diff
@@ -0,0 +1,41 @@
+## oneview_interconnect_facts
+Retrieve facts about one or more of the OneView Interconnects.
+
+#### Synopsis
+ Retrieve facts about one or more of the Interconnects from OneView.
+
+#### Requirements (on the host that executes the module)
+  * python >= 2.7.9
+  * hpeOneView >= 5.4.0
+
+#### Options
+
+| Parameter     | Required    | Default  | Choices    | Comments |
+| ------------- |-------------| ---------|----------- |--------- |
+| config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
+| name  |   No  |  | |  Interconnect name.  |
+| options  |   No  |  | |  List with options to gather additional facts about Interconnect. Options allowed: `nameServers` gets the named servers for an interconnect. `statistics` gets the statistics from an interconnect. `portStatistics` gets the statistics for the specified port name on an interconnect. `subPortStatistics` gets the subport statistics on an interconnect. `ports` gets all interconnect ports. `port` gets a specific interconnect port. `pluggableModuleInformation` gets all the SFP information.  To gather additional facts it is required inform the Interconnect name. Otherwise, these options will be ignored.  |
```

Comment:
please change this. This info mentions about interconnects resource.

Reply:
done